### PR TITLE
Add new workload app echo-plus for e2e test

### PR DESCRIPTION
**Pull request by songrx1997**
/assign @swetharepakula

### CODE REVIEW

File: cmd/echo-plus/app/flags.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		RequestLatency      int
+	}{}
+)
+
+// RegisterFlags creates flags.
+func RegisterFlags() {
+	flag.DurationVar(&F.CertificateLifeSpan, "cert-duration", DefaultCertLifeSpan, "The lifespan of the TLS certificate created on binary start")
+	flag.IntVar(&F.HTTPPort, "http-port", 8080, "Port use for HTTP, 0 will disable this protocol")
+	flag.IntVar(&F.HTTPSPort, "https-port", 8443, "Port use for HTTPS, 0 will disable this protocol")
+	flag.IntVar(&F.RequestLatency, "reqeust-latency", 10000, "Time of reqeust latency for /slowRequest (in ms)")
```
Comment (swetharepakula) [inline cmd/echo-plus/app/flags.go:42]: `Maximum latency for /randomSlowRequest`.

/slowRequest I thought uses the value found in the path.

File: cmd/echo-plus/app/echoplus_test.go [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+		},
+		{
+			desc:           "Test slowRequest with wrong path",
+			path:           "/slowRequest/abc",
+			method:         http.MethodGet,
+			statusExpected: http.StatusBadRequest,
+		},
+		{
+			desc:           "Test slowRequest with right path",
+			path:           "/slowRequest/10",
```
Comment (swetharepakula) [inline cmd/echo-plus/app/echoplus_test.go:35]: Maybe do 1 sec, so the tests don't take too long to run.

**Review by swetharepakula — COMMENTED** [review]
A couple of small nits, otherwise LGTM. Thanks Ruixian!

File: cmd/echo-plus/app/flags.go (hunk 1 above)
Comment (songrx1997) [reply cmd/echo-plus/app/flags.go]: Updated this to `/randomSlowRequest`

File: cmd/echo-plus/app/echoplus_test.go (hunk 2 above)
Comment (songrx1997) [reply cmd/echo-plus/app/echoplus_test.go]: Updated the time to 1s

**Pull request closed without merging**